.github/workflows/check-contracts-and-assets.yml

```diff
@@ -65,4 +95,25 @@ jobs:
             # a multiline string
             data-asset-options: |
               --source-type avro \
-              --files ./event_schemas/*.avsc
\ No newline at end of file
+              --files ./event_schemas/*.avsc
+      - name: Check Postgres Data Assets
+        uses: gabledata/cicd/github-actions/check-data-assets@latest
+        with:
+            # Provide API key secret, and endpoint variable created in previous steps
+            gable-api-key: ${{secrets.GABLE_API_KEY}}
+            gable-api-endpoint: ${{secrets.GABLE_API_ENDPOINT}}
+            # List of paths to Avro files that should be checked with support for glob syntax. 
+            # Can either be specified as a space separated list ('event1.proto event2.proto'), or 
+            # a multiline string
+            data-asset-options: |
+              --source-type postgres \
+              -h prod.store.com \
```

Comment:
I think we should use the full option names here just so it's clear what's going on, especially for the proxy database options below